eq-author-api/yarn.lock

```diff
@@ -991,10 +991,11 @@ apollo-link@^1.2.3:
     zen-observable-ts "^0.8.19"
 
 apollo-opentracing@latest:
-  version "1.4.1"
-  resolved "https://registry.yarnpkg.com/apollo-opentracing/-/apollo-opentracing-1.4.1.tgz#878781ed8e8abf3b257122991fd55b5f1e219ea9"
-  integrity sha512-UrRvEnXQMoV8StdRyYiVEkJoOCviO6O9xCZGwPVRxLsj3VlwmFr3yeaf2M/PuJgwoEXq7h2SX0JmMLrm/EUdfA==
+  version "1.4.2"
+  resolved "https://registry.yarnpkg.com/apollo-opentracing/-/apollo-opentracing-1.4.2.tgz#2c96af989685a4a72a173e683b8dd11fe7c79b31"
+  integrity sha512-5IgBIixqqmWWOo+TAsPYVRq9JbUgoPnRLckutgIgFpoSVxhM78nkAmwuXBF8lRveXWy113GnDEihAJ2nuOmF2Q==
```

Comment:
is package update necessary here? I suspect this has already been updated too.

Reply:
caused by rebase